### PR TITLE
Chapter 5: Filling in the layout

### PR DESCRIPTION
**Pull request by longcn-1045**
Chapter 5: Filling in the layout

### CODE REVIEW

**Comment by longcn-1045** [comment]
Screenshot from 2020-08-07 14-51-16

**Comment by longcn-1045** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -0,0 +1,101 @@
+@import "bootstrap";
```
Comment (manhbnt-0960) [inline app/assets/stylesheets/custom.scss:1]: dùng '

File: app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (manhbnt-0960) [inline app/assets/stylesheets/users.scss:1]: xóa file k dùng

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (manhbnt-0960) [inline app/helpers/users_helper.rb:1]: xóa file k dùng

File: app/views/layouts/_footer.html.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  <nav>
+  <ul>
+    <li><%= link_to t(".about"), "#" %></li>
+    <li><%= link_to t(".contact"), "#" %></li>
+    <li><%= link_to t(".news"), "#" %></li>
+  </ul>
+  </nav>
+</footer>
+
+
```
Comment (manhbnt-0960) [inline app/views/layouts/_footer.html.erb:15]: xóa dòng trống

File: app/views/users/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<% provide(:title, "Sign up") %>
```
Comment (manhbnt-0960) [inline app/views/users/new.html.erb:1]: i18n

File: app/assets/stylesheets/custom.scss (hunk 1 above)
Comment (longcn-1045) [reply app/assets/stylesheets/custom.scss:1]: vang a

File: app/assets/stylesheets/users.scss (hunk 2 above)
Comment (longcn-1045) [reply app/assets/stylesheets/users.scss:1]: vang a

File: app/helpers/users_helper.rb (hunk 3 above)
Comment (longcn-1045) [reply app/helpers/users_helper.rb:1]: vang a

File: app/views/layouts/_footer.html.erb (hunk 4 above)
Comment (longcn-1045) [reply app/views/layouts/_footer.html.erb:15]: oki a

File: app/views/users/new.html.erb (hunk 5 above)
Comment (longcn-1045) [reply app/views/users/new.html.erb:1]: vang a

**Comment by longcn-1045** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+      float: right;
+      list-style: none;
+      li {
+        float: left;
+        margin-left: 15px;
+      }
+    }
+  }
+}
+
```
Comment (manhbnt-0960) [inline app/assets/stylesheets/custom.scss:101]: xóa dòng tróng

Comment (longcn-1045) [reply app/assets/stylesheets/custom.scss]: ok a

**Pull request merged**